src/status_im/ethereum/encode.cljs

```diff
@@ -0,0 +1,8 @@
+(ns status-im.ethereum.encode
+  (:require [status-im.js-dependencies :as dependencies]))
+
+(defn utils [] (dependencies/web3-utils))
+
+(defn uint
+  [x]
+  (.numberToHex (utils) x))
```

Comment:
Will memoize for `utils` have any impact here?

Reply:
hard to tell I think it is ok for now, also we are discussing with @dshulyak the idea of receiving transit encoded data and status-go would do the conversion, but I tested on an account with 200 transfers and it doesn't seem to be an issue